### PR TITLE
fix(mulesoft): replaced all mulesoft references with APIM

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Remove all the references to Mulesoft.

## Resolution
* Replaced all `MULESOFT_*` references with `APIM_MDM`.
* Introduced APIM respective constants.

### CODE REVIEW

File: src/ui/server/api/external/index.ts [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
 dotenv.config();
 
+const { APIM_MDM } = EXTERNAL_API_ENDPOINTS;
 const headers = {
   'Content-Type': 'application/json',
   [String(process.env.APIM_MDM_KEY)]: process.env.APIM_MDM_VALUE,
 };
 
 const external = {
+  getCountries: async () => {
```
Comment (ttbarnes) [inline src/ui/server/api/external/index.ts:14]: This is not valid, this has been replaced with a UI call to our own API, which then calls APIM CIS endpoint. Do you have the latest main-application`?

File: src/ui/server/constants/external-apis.ts [hunk 2]
```diff
@@ -9,3 +9,11 @@ export const EXTERNAL_API_MAPPINGS = {
...
   },
 };
+
+export const EXTERNAL_API_ENDPOINTS = {
+  APIM_MDM: {
+    CURRENCY: '/currencies',
+    INDUSTRY_SECTORS: '/sector-industries',
+    MARKETS: '/markets',
+  },
+};
```
Comment (ttbarnes) [inline src/ui/server/constants/external-apis.ts:19]: Same as above, this has been removed

**Review by ttbarnes — CHANGES_REQUESTED** [review]
Great work just a few comments

File: src/ui/server/api/external/index.ts (hunk 1 above)
Comment (abhi-markan) [reply src/ui/server/api/external/index.ts:14]: Have removed.

File: src/ui/server/constants/external-apis.ts (hunk 2 above)
Comment (abhi-markan) [reply src/ui/server/constants/external-apis.ts]: Have removed.

File: src/ui/server/api/external/index.ts [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
+      console.error('Error getting CIS countries %O', err);
+      throw new Error(`Getting CIS countries ${err}`);
+    }
+  },
   getCurrencies: async () => {
     try {
       const config: RawAxiosRequestConfig = {
         method: 'GET',
-        url: `${process.env.APIM_MDM_URL}/currencies`,
+        url: `${process.env.APIM_MDM_URL}/${APIM_MDM.CURRENCY}`,
```
[the remark this answers is not in the document]
Comment (abhi-markan) [reply src/ui/server/api/external/index.ts]: Understood, have reverted back.

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**